homeassistant/components/freebox/sensor.py

```diff
@@ -11,10 +11,52 @@
 async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
     """Set up the sensors."""
     fbx = hass.data[DATA_FREEBOX]
-    async_add_entities([FbxRXSensor(fbx), FbxTXSensor(fbx)], True)
+    sys_config = await fbx.system.get_config()
+    fbx_sensors = []
+    for temp_sensor_id in sys_config["sensors"]:
+        fbx_sensors.append(FbxTempSensor(fbx, temp_sensor_id["id"]))
 
+    fbx_sensors.extend([FbxRXSensor(fbx), FbxTXSensor(fbx)])
+    async_add_entities(fbx_sensors, True)
 
-class FbxSensor(Entity):
+
+class FbxTempSensor(Entity):
+    """Representation of a temperature sensor."""
+
+    def __init__(self, fbx, sensor_id):
+        """Initialize the sensor."""
+        self._fbx = fbx
+        self._datas = None
+        self._name = "generic"
+        self._state = None
+        self._sensor_id = sensor_id
+        self._unit_of_measurement = "°C"
```

Comment:
Please import the following constant and use it here, thanks :)
https://github.com/home-assistant/home-assistant/blob/b4caa4ab82012652448b1130ca8ac43fd0201689/homeassistant/const.py#L338

Reply:
Thx !